esp/platform/espcontext.cpp

```diff
@@ -1020,6 +1020,53 @@ unsigned getSlowProcessingTime()
     return false;
 }
 
+LogMsgDetail mapLegacyEspLogLevelToJlogThreshold(LogLevel level)
+{
+    if (level == LogNone)
+    {
+        return(LogNoneLegacyThreshold);
+    }
+    else if (level >= LogMin && level < LogNormal)
+    {
+        return(LogMinLegacyThreshold);
+    }
+    else if (level >= LogNormal && level < LogMax)
+    {
+        return(LogNormalLegacyThreshold);
+    }
+    return(LogMaxLegacyThreshold);
+}
+
+LogMsgCategory getEspLogCategoryForContext(IEspContext* ctx, LogMsgCategory defaultLogMsgCategory)
+{
+    if (ctx)
+    {
+        ISecPropertyList* properties = ctx->querySecuritySettings();
```

Comment:
This would be cleaner encapsulated as a function ctx->includeDebugLogging() or similar.  I'm not clear what the function could be called because I'm not sure what it is logically testing.